scripts/cleanup_empty_annotations.py

```
@@ -0,0 +1,142 @@
+
+    if response.status_code != 200:
+        return []
+
+    return response.json()
+
+
+def delete_annotation(token: str, annotation_id: int) -> bool:
+    """Delete an annotation."""
+    headers = {"Authorization": f"Bearer {token}"}
+
+    response = requests.delete(
```

Ruff (0.14.7)

Probable use of `requests` call without timeout

(S113)